modules/storage-ephemeral-vols-scc-bug.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * storage/generic-ephemeral-vols.adoc
+
+:_content-type: PROCEDURE
+[id="generic-ephemeral-vols-scc-bug_{context}"]
+= Creating a custom SCC to allow use of generic ephemeral volumes
+
+The default security context constraint (SCC) may cause an Amazon Web Services (AWS) cluster deployment using generic ephemeral volumes to remain in the pending state.
```

Comment:
Hi @lpettyjo The SCC issue workround is a general way for all cloudproviders clusters, not only aws has the limation. In addition, from my understanding, we'd better move the steps before create the generic ephemeral volumes if possible. Others looks good to me. Thanks for help update!

Reply:
@Phaow Thanks! Changes made.